crates/client/assets/schema.sdl

```diff
@@ -555,6 +555,16 @@ type MessageProof {
 	data: HexString!
 }
 
+enum MessageState {
+	UNSPENT
+	SPENT
+	UNKNOWN
```

Comment:
I think I prefer `UNKNOWN` over `NOTBRIDGED`, because it might not just show up in my DB yet.

I guess `UNKNOWN` is too ambiguous?

How about `NOTFOUND`?

Reply:
`NOTFOUND` is good=)